### ISSUE TITLE
Hello Tim

### ISSUE THREAD

**Issue by sidPhoenix17** [theurichde/theurichde#1]
Hi Tim,
I've never communicated with anyone in the past via a Github issue, definitely my first one! :D 

I read about how you folks implemented Backstage and TechDocs at Idealo -- loved it.

I'm Siddarth -- Founder of Doctor Droid -- we are building an assistant tool that speeds up troubleshooting and investigation for developers on-call.

Here's a short note on our [product](http://feedback.drdroid.io/b?y=49q24eh26ssjge31clgm6d1i6csmcdhlcgpm6d9jcks38dhlcos3epj4cch2o8ji48t24q3keho76ehf5ti6uorj5pi74p3idtkm8bj9dsnm8rr3ecnm2tbkdtmm2t355lkmqs31cdq2qobjedin6srdcln78bb4d5gmerjfedkn68g=). 

tl;dr Replace on-call docs with a LIVE investigation playbook which fetches relevant data (plots / prints them) from user's existing observability tools and helps instantly eliminate obvious root causes. 

Would you be open to joining our beta? 

Let me know your thoughts!

**Comment by theurichde** [comment]
Hi. Thanks for reaching out - my first comm via GitHub issue, too 😅 

 > Would you be open to joining our beta?

 Nope Sir, not interested at this point.

Bye!
